app/Http/Controllers/Api/Device/DeviceGraphController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Http\Controllers\Api\Device;
+
+use App\Models\Device;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Api\ApiController;
+
+class DeviceGraphController extends ApiController
+{
+    /**
+     * @api {get} /devices/:id/graphs Get Generic graphs
```

Comment:
This indicates it gets graphs. Maybe `/devices/:id/graphs/available`?

Reply:
This could probably be renamed to generic or something?